apps/coa/behaviours/save-desired-content.js

```diff
@@ -0,0 +1,18 @@
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    // nameWithPossession returns the name which puts an ('s) on the end of the applicant's name.
+    // If the name ends with S, its just returns the applicant's name with apostrophe (') at end.
+    const applicantName = req.sessionModel.get('applicant-full-name');
+    const nameWithPossession = applicantName + (applicantName.toLowerCase().endsWith('s') ? '’' : '’s');
```

Comment:
tbc with UDC team: Perhaps we can skip this step, as grammatically it would be correct to use `'s` all the time 
https://dictionary.cambridge.org/grammar/british-grammar/possession-john-s-car-a-friend-of-mine#possession-john-s-car-a-friend-of-mine__3:~:text=When%20a%20first,%CB%88ki%3Ats%C9%AAz/)

Reply:
Confirmed with UCD team: It is preferable to have this logic, which adds an apostrophe S to the end of the applicant's name, depending on whether it ends in S or not, otherwise it will just have an apostrophe.